### ISSUE TITLE
 Symfony 7 support.

### ISSUE THREAD

**Issue by laurentmuller** [createnl/zxcvbn-bundle#15]
The Symfony version 7 is out. It will be nice to update the dependencies to be able to use this version.

It is just mater to update some lines in the `composer.json`:

```json
    "require": {
        "symfony/http-kernel": "^6.0|^7.0",
    }
    
    "require-dev": {
        "symfony/framework-bundle": "^5.4|^6.0|^7.0",
        "symfony/console": "^5.4|^6.0|^7.0",
    }
```
I can create a PR if You want.

**Comment by AlexLisenkov** [comment]
Fixed in release 3.1.0
Thank you for the pull request

**Comment by laurentmuller** [comment]
You'r welcome...
